compiler/rustc_resolve/src/imports.rs

```diff
@@ -554,6 +555,40 @@ impl<'a, 'tcx> Resolver<'a, 'tcx> {
         }
     }
 
+    pub(crate) fn check_local_binding_shadows_glob_reexport(&mut self) {
+        for module in self.arenas.local_modules().iter() {
+            let resolutions = self.resolutions(module).borrow();
+            for (_, resolution) in resolutions.iter() {
+                let resolution = resolution.borrow();
+                if let Some(binding) = resolution.binding
+                && let Some(glob_binding) = resolution.shadowed_glob
```

Comment:
style nit: let chains are indented, but you don't have to fix that if you don't want to as long as rustfmt doesn't support it